projects/aca-content/assets/app.extensions.json

```diff
@@ -2076,6 +2076,17 @@
           "sortable": true,
           "desktopOnly": true,
           "order": 50
+        },
+        {
+          "id": "app.filer.tags",
+          "template": "app.columns.tags",
+          "key": "tags",
+          "title": "APP.DOCUMENT_LIST.COLUMNS.TAGS",
+          "class": "adf-full-width adf-expand-cell-4",
+          "type": "text",
+          "sortable": false,
+          "desktopOnly": false,
```

Comment:
you may want to put true for smaller screens.... what it does - hides the column on the small screens

Reply:
Yeah I thought about it as well, I'll consult it with Lilian and Chapman to be sure if we want to hide it

Reply:
Lilian confirmed that we want to hide tags for smaller displays, changed to `true` now